scripts/migration/migrate_date_modified_for_existing_nodes.py

```diff
@@ -25,7 +26,7 @@ def main():
 
     logger.warn('Date_modified field will be added to all nodes.')
 
-    for node in models.Node.find():
+    for node in models.Node.find(Q('date_modified', 'eq', None)):
```

Comment:
I don't understand what this query adds.  I have four nodes, one has date_modified as a field, the others do not.  If I do `Node.find(Q('date_modified', 'eq', None))`, three nodes are returned, and if I do `Node.find(Q('date_modified', 'exists', False))`, three nodes are also returned.